netket/operator/_sumoperators.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional, List
+
+from netket.utils.types import DType
+
+from netket.hilbert import AbstractHilbert
+from netket.operator import ContinousOperator
+
+import jax.numpy as jnp
+
+
+class SumOperator(ContinousOperator):
```

Comment:
nice

Reply:
@gpescia missing docstring describing this class